### PR TITLE
Protegrity integration template refactoring

### PR DESCRIPTION
**Pull request by Nuzhdina-Elena**
Refactor and format pipeline code

### CODE REVIEW

File: v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java [hunk 1]
```diff
@@ -260,54 +266,50 @@ private POutput writeAvro(PCollection<Row> outputCollection, Schema schema) {
...
+              .apply(
+                  "WriteCsvConversionErrorsToGcs",
                   ErrorConverters.WriteErrorsToTextIO.<String, String>newBuilder()
                       .setErrorWritePath(options.getNonTokenizedDeadLetterGcsPath())
                       .setTranslateFunction(SerializableFunctions.getCsvErrorConverter())
                       .build());
         }
-        /*
-         * Step 4: Get jsons that were successfully processed.
-         */
```
Comment (KhaninArtur) [inline v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java]: I think comments for these 4 steps should stay - they nicely explain what's going on

File: v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java [hunk 2]
```diff
@@ -260,54 +266,50 @@ private POutput writeAvro(PCollection<Row> outputCollection, Schema schema) {
...
         PCollectionTuple csvLines = readCsv(pipeline);
-        /*
-         * Step 2: Convert lines to Json.
-         */
         PCollectionTuple jsons = csvLineToJson(csvLines, schema.getJsonBeamSchema());
 
         if (options.getNonTokenizedDeadLetterGcsPath() != null) {
-          /*
-           * Step 3: Write jsons to dead-letter gcs that were successfully processed.
```
Comment (KhaninArtur) [inline v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java:285]: Please replace `were successfully` with `were not successfully`

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java (hunk 1 above)
Comment (Nuzhdina-Elena) [reply v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java]: Done. But I think names of methods say about current step :)

File: v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java (hunk 2 above)
Comment (Nuzhdina-Elena) [reply v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java:285]: Done.

**Review by KhaninArtur — APPROVED** [review]
LGTM
@ilya-kozyrev, please take a look

**Pull request merged**